README.md

```diff
@@ -1 +1,74 @@
 # canvas_courses_report
+
+## Build Directions
+ 
+1. CourseReportTool$ <code>mvn clean install</code>
+2. Copy to tomcat/webapp
+3. Add the property file on linux box <code>coursereport.properties</code>,   
+then in JAVA_OPTS add the  
+<code>-DcanvasCourseReportPath=file:/file-path/coursereport.properties</code><br/>
+
+
+4. Add the following 5 properties to coursereport.properties: 
+
+    <code>canvas.admin.token=canvas token  
+    canvas.url=target canvas server e.g. https://umich.test.instructure.com  
+    use.test.url=true  
+    ldap.server.url=ldap server e.g. ldap://ldap.itd.umich.edu:389/dc=umich,dc=edu  
+    mcomm.group=group that can use this tool e.g. its-canvas-reports</code>
```

Comment:
The formatting for this block of code is wrong.  See the GH rendering of this document to see how it looks:

https://github.com/tl-its-umich-edu/canvas_courses_report/blob/TLUNIZIN-683/README.md

Use the same kind of formatting that was used for steps six or seven, below, which appear to be correct.  In general, avoid the HTML-like tags in Markdown documents.  Use the accent mark instead.